modules/osdk-cli-reference-olm-install.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-olm-install_{context}"]
+= install
+`olm install` subcommand installs OLM in your cluster.
```

Comment:
Please add a blank line between the heading and the first paragraph.  I saw a few other instances, please revise globally and thank you.